dockstore-common/src/test/java/io/dockstore/common/PipHelperTest.java

```diff
@@ -3,8 +3,6 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import static org.junit.Assert.*;
```

Comment:
Good find, please create a ticket for 1.8.0 to apply checkstyle.xml to tests already